docs/Demo.md

```diff
@@ -1,73 +1,175 @@
+> **Note:**
+> Ensure that your in your github fork of mig-demo-apps is correctly set as the uri value in the todolist BuildConfig.
+```
+- kind: BuildConfig
+  apiVersion: build.openshift.io/v1
+  metadata:
+    name: todolist
+<snip>
+      source:
+        type: Git
+        git:
+          uri: https://github.com/weshayutin/mig-demo-apps.git
+          ref: master
 
-The second argument the script takes is the url of the forked repo, e.g. `https://github.com/kenwilli/container-pipelines.git`
+```
 
 ## Flow
-- Deploy sample application (basic-nginx)
-- Make changes to the application (adding a line to index.html)
+- Make changes to the application, e.g. replace a line to index.html
 - commit changes to source control
-- redeploy application with the changes to be captured by pelorus
+- Watch the application redeploy with the changes to be captured by Pelorus
 
-## Execution
+## Github Webhook
 
-An "idle" state could resemble:
-![Idle-Data](img/pelorus-dashboard-idle-data.png)
+You can more easily see how Pelorus works by automatically building and deploying the todolist app if you have configured your source git repository webhooks.
+
+1. To get the build webhook URL you can navigate to the todolist BuildConfig details or via the cli:
+```
+oc get all -n mongo-persistent
+oc describe buildconfig.build.openshift.io/todolist -n mongo-persistent
+```
+> **Note:**
+> The secret is hardcoded in the todolist manifest template to be:
+```
+4Xwu0tyAab90aaoasd88qweAasdaqvjknfrl3qwpo
+```
+* Navigate to https://github.com/your_org/mig-demo-apps/settings/hooks
+* Paste the URL with the real secret replacing the text <secret>
+* Toggle SSL as needed, for testing consider disabling.
+* Content type: application/json
+* Click, "Add webhook"
+
+## Update the application source
+The following screenshot is the original todolist application prior to a change
+
+![todolist](img/todolist_orig.png)
+
+* The text "Enter an activity" does not seem clear, let's change that to "Add a todo"
+
+```
+cd mig-demo-apps/apps/todolist-mongo-go
+sed -i 's/Enter an activity../Add a todo../g' index.html
+```
+
+* If you are happy with the change, commit and push
+```
+git add .
+git commit -m "update text box"
+git push origin master
+```
+
+### Rebuilding and Deploying the todolist application
+* Once the commit is pushed the application will automatically rebuild because we have setup the github webhook.
+
+* You will now see the todolist application start to rebuild
+![todolist_rebuild](img/todolist_rebuild.png)
+
+### Understand the changes to the Grafana Dashboard
 
-> **Note**  
-> Dashboard can be found by going to grafana which url can found with:  
-> `oc get route grafana-route -o json | jq -r '.spec.host'`  
+> **Note:**
+> Dashboard can be found by going to grafana which url can found with:
+> `oc get route grafana-route -o json | jq -r '.spec.host'`
```

Comment:
This will not work, as the namespace is not defined.
`oc get route grafana-route -o json -n pelorus | jq -r '.spec.host'`